src/NHibernate.Test/GhostProperty/GhostPropertyDynamicProxyFixture.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NHibernate.Bytecode;
+using NUnit.Framework;
+
+
+namespace NHibernate.Test.GhostProperty
+{
+	// Since 5.3
+	[Obsolete]
+	[TestFixture]
+	public class GhostPropertyDynamicProxyFixture : GhostPropertyFixture
+	{
+		protected override void Configure(Cfg.Configuration configuration)
+		{
+			base.Configure(configuration);
+			configuration.SetProperty(Cfg.Environment.ProxyFactoryFactoryClass, typeof(DefaultProxyFactoryFactory).FullName);
```

Comment:
For being future proof, better save here `Cfg.Environment.BytecodeProvider.ProxyFactoryFactory.GetType().FullName` in some field, and use that to restore it after the test.